spec/datetimeformat.html

```diff
@@ -732,6 +752,7 @@ <h1>Properties of Intl.DateTimeFormat Instances</h1>
       <li>[[NumberingSystem]] is a String value with the `"type"` given in Unicode Technical Standard 35 for the numbering system used for formatting.</li>
       <li>[[TimeZone]] is a String value with the IANA time zone name of the time zone used for formatting.</li>
       <li>[[Weekday]], [[Era]], [[Year]], [[Month]], [[Day]], [[Hour]], [[Minute]], [[Second]], [[TimeZoneName]] are each either *undefined*, indicating that the component is not used for formatting, or one of the String values given in <emu-xref href="#table-datetimeformat-components"></emu-xref>, indicating how the component should be presented in the formatted output.</li>
+      <li>[[MillisecondDigits]] is a non-negative integer Number value indicating the fraction digits to be used for franctional second. Numbers will be rounded or padded with trailing zeroes if necessary. </li>
```

Comment:
>  are each either *undefined*, indicating that the component is not used for formatting

It there a reason that this part is missing for the new `MillisecondDigits`?

Reply:
In the current PR, MillisecondDigits is 
A) an integer, range from 0 to 3. When the value is 0, it means there are 0 digits to be output. We can also choose to define it is 
B) an integer in the range from 1 to 3 plus undefined. 
We should choose between A) and B), I do not believe B) is better than A). But if you think B) is better than A), could you explain why?

Reply:
Oh, you are right, thanks.
`MillisecondDigits` is an integer and not an enum, therefor we have zero instead of `undefined` as a default. 
I was not familiar with `GetNumberOption`, so i missed the fallback part. Sorry for the noise.

Reply:
NP